src/data/reports/data-quality/DataQualityDefaultRepository.ts

```diff
@@ -167,29 +167,25 @@ export class DataQualityDefaultRepository implements DataQualityRepository {
         api: D2Api,
         metadataItems: IndicatorMetadataType[]
     ): Promise<IndicatorItem[]> {
-        return _.compact(
-            await promiseMap(metadataItems, async item => {
-                try {
-                    const numeratorValidation = await api.expressions.validate("indicator", item.numerator).getData();
-                    const denominatorValidation = await api.expressions
-                        .validate("indicator", item.denominator)
-                        .getData();
-
-                    const numeratorResult = this.statusToBoolean(numeratorValidation.status);
-                    const denominatorResult = this.statusToBoolean(denominatorValidation.status);
-
-                    return {
-                        ...item,
-                        user: item.user.displayName,
-                        numeratorResult,
-                        denominatorResult,
-                        metadataType: "Indicator",
-                    };
-                } catch (error) {
-                    console.debug(error);
-                }
-            })
-        ) as IndicatorItem[];
+        return (await promiseMap(metadataItems, async item => {
```

Comment:
This is not related to GLASS, right?

Reply:
Oh, ignore this comment, I just saw that the following commit is a revert of this

Reply:
no. i will revert the changes here